srcpkgs/ibus-libpinyin/template

```diff
@@ -0,0 +1,18 @@
+# Template file for 'ibus-libpinyin'
+pkgname=ibus-libpinyin
+version=1.10.92
+revision=1
+build_style=gnu-configure
+configure_args="--prefix=/usr"
+hostmakedepends="intltool libtool pkg-config autoconf-archive autoconf automake"
+makedepends="gnome-common sqlite-devel ibus-devel glib-devel gettext-devel libpinyin"
```

Comment:
gnome-common and gettext-devel are more commonly on hostmakedepends=

repalce `glib-devel` with `libglib-devel`

create a libpinyin-devel package and use it.

Reply:
Can the subpkg be installed and referenced by another plugin? I believe the libpinyin library is also used by ibus-pinyin, which is yet another implementation of the Chinese input engine with slight difference (I believe it has one more layer of lookup/smarts).

Reply:
@maxice8  I read the manual again. My question is answered (re: subpackages being able to be referenced from another template - the symlink solves that issue). What's the best practice/convention? Always create subpkg when we can? Or split the libfoo-devel package out when we know other people might use it?